src/redux/books/bookSlice.js

```diff
@@ -12,10 +31,10 @@ const booksSlice = createSlice({
       ...state,
       books: [...state.books, action.payload],
     }),
-    removeBook: (state, action) => ({
-      ...state,
-      books: state.books.filter((book) => book.id !== action.payload),
-    }),
+    removeBook: (state, action) => {
+      const index = state.books.findIndex((book) => book.item_id === action.payload.id);
+      state.books.splice(index, 1);
```

Comment:
- [ ] You did a great job at adding action, but I see that there is an issue with removing action as when I click on the remove button from up I see that the specific item to be removed and the next item removed, so please try to fix that to make the specific item removing 
- [ ] Also I 😃 see that there is a remove button exits when I remove all items I don't know this is because what although there is no items, so please try to fix that when there is no items the page will not have buttons as no items in the page 💯👌


> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| My design | Your design| 
| --- | --- |
|  ![Recording 2023-04-05 at 14 00 34 (1)](https://user-images.githubusercontent.com/83360636/230074303-e299abc4-a1d1-4710-adbc-0b59671a41f1.gif)   |   ![image](https://user-images.githubusercontent.com/83360636/230073712-629c81c7-b2da-4289-93c7-2fcea50c4429.png)    |